tuxemon/locale.py

```diff
@@ -247,29 +251,62 @@ def replace_text(session: Session, text: str) -> str:
 
     for i in range(len(player.monsters)):
         monster = player.monsters[i]
-        text = text.replace("${{monster_" + str(i) + "_name}}", monster.name)
-        text = text.replace(
-            "${{monster_" + str(i) + "_desc}}",
-            monster.description,
-        )
-        text = text.replace("${{monster_" + str(i) + "_type}}", monster.slug)
-        text = text.replace(
-            "${{monster_" + str(i) + "_category}}",
-            monster.category,
-        )
-        text = text.replace("${{monster_" + str(i) + "_shape}}", monster.shape)
-        text = text.replace(
-            "${{monster_" + str(i) + "_hp}}",
-            str(monster.current_hp),
-        )
-        text = text.replace(
-            "${{monster_" + str(i) + "_hp_max}}",
-            str(monster.hp),
-        )
-        text = text.replace(
-            "${{monster_" + str(i) + "_level}}",
-            str(monster.level),
-        )
+        _name = monster.name
+        text = text.replace("${{monster_" + str(i) + "_name}}", _name)
+        _description = monster.description
+        text = text.replace("${{monster_" + str(i) + "_desc}}", _description)
+        _types = ""
+        for _type in monster.types:
+            _types += T.translate(_type.name) + " - "
+        text = text.replace("${{monster_" + str(i) + "_types}}", _types[:-3])
+        _category = monster.category
+        text = text.replace("${{monster_" + str(i) + "_category}}", _category)
```

Comment:
I think i wasn't being clear, sorry about that.
This line should just be

`text = text.replace("${{monster_" + str(i) + "_category}}", monster.category)`

Same with all the other lines in this section. There's no point in saving the value, except in the case of types, since there's multiple.

Reply:
Fine, but it'll be longer the code since it goes over the 79 character per line